### ISSUE TITLE
Upload Package

### ISSUE THREAD

**Issue by kaaax0815** [cdr/code-server#2600]
Upload Package to install it easily with `apt install code-server`

**Comment by nhooyr** [comment]
Originally we determined it'd be too much effort to get code-server into all these upstream repositories. The install script + github releases has been very streamlined for us but there's definitely value in users being able to install directly from their distro vendor.

**Comment by kaaax0815** [comment]
I think this (https://askubuntu.com/questions/16446/how-to-get-my-software-into-ubuntu) is useful for beginning or discussing if you want a PPA or the Official Ubuntu repo

**Comment by tidux** [comment]
Given the release cadence of code-server versus Debian/Ubuntu I think it'd be better to just stand up a third party apt repo.  The current .debs work fine.

**Comment by kaaax0815** [comment]
This would also be a good alternative, because this would also fix https://github.com/cdr/code-server/issues/2582

**Comment by KaKi87** [comment]
+1 for adding `code-server` to the main Debian/Ubuntu APT repository, or to create a PPA.
DEBs are already very nice, but doesn't allow automated updates.
Unless it would be preferred to code a self-updater but I don't think it would be easier.
THanks

**Comment by tianalemesle** [comment]
up

**Comment by KaKi87** [comment]
Hi @code-asher, any news on an APT repo or a PPA or a flatpak ? Thanks

**Comment by code-asher** [comment]
No, I have no plans on doing this, but if someone wants to set something up that automatically runs with the CI publish workflow or something, please feel free.

**Comment by KaKi87** [comment]
Hi,

A few months ago, I started developing a tool named [dynapt](https://git.kaki87.net/KaKi87/dynapt) to automatically fetch updates through APT for packages that don't provide an APT repository.

As I've been upgrading code-server through it for quite some time now, I just published my first pre-release of it, which you can download and set up from [this tutorial](https://gist.github.com/KaKi87/f620788e9901abbfef4978eb7ad358b4).

For code-server specifically, replace the following configuration line :

```json
    "apps": []
```

With the following :

```json
    "apps": [
        {
            "name": "code-server",
            "github": {
                "repo": "coder/code-server",
                "filter": "*amd64*"
            }
        }
    ]
```

But also change `amd64` to `arm64` if applicable.

For questions or issues, submit a comment on the tutorial page.

Thanks
